game/song.hh

```diff
@@ -134,5 +135,7 @@ private:
 	bool m_silent;
 };
 
+using SongPtr = std::shared_ptr<Song>;
```

Comment:
looks really useless here, never used in this file nor cc

Reply:
Should be used instead all the std::shared_ptr<Song>, so I think it is a good place to define it.